content/blog/qmstr-milestone-3-report.md

```diff
@@ -0,0 +1,150 @@
++++
+title = "Quartermaster Milestone 3 Development Report: C++, Linux, CCache"
+date = "2019-02-03T00:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance", "milestone", "report"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-ms-3-report-title.jpg"
++++
+
+Version 0.3 of Quartermaster was tagged on February 1, 2019. It
+includes improvements to allow for example the Linux kernel, glibc and
+openssl as projects under analysis. It delivers support for snapshots
+of the knowledge graph, which allows to roll back changes to a known
+state, as well as support for source code in assembly language. It
+improves the support for `ccache`, `ar`, `ld` and `objcopy`, and for
+analysing source code elements that are generated during the build and
+are not part of the original source code package. Quartermaster is
+Free and Open Source software and developed under a collaborative open
+governance model. As usual, the source code is available on
+[Github](https://github.com/QMSTR/qmstr/releases/tag/v0.3). Read more
+for all the details on the new release.  
+<!--more-->
+
+# Quartermaster Version 0.3
+
+The new release marks one year of Quartermaster development. The
+development team focused on covering complete use cases of real-life
+Open Source projects like the Linux kernel, openssl or glibc. This
+approach helped to identify gaps in Quartermaster functionality which
+led, for example, to added functionality for `ld` and `objcopy` that
+are used in the build system of these projects. One concern in such
+scenarios is that code gets either generated or combined from other
+elements in the source tree and then passed to the compiler via stdin,
+without being saved to the file system. In such a case it is difficult
+to identify what exact code is compiled and added to the resulting
+binary. The code is now saved in the knowledge graph for inspection.
```

Comment:
It is actually not saved in the graph but in the file system to get checked by e.g. scancode